### DISCUSSION TITLE
Folder Structure

### DISCUSSION THREAD

**Discussion by FdezAriasSara** [Arquisoft/lomap_en3b#47] [Ideas]
Hi ~everyone sorry for so much activity, but we need to decide the folder structure of the project.
 
I suggest the following:

-components folder: It will contain all elements that can be reusable : for example, Headers , sidebars...
-views : Will contain specific frontend views: LoginView, FriendsView....
-handlers : classes related to business logic
-models: files containing the classes of the objects of the app
-util : any utilities like data parsers, checks.. that can be used all over the app

I also changed the name userView to mapView in my proposal  so its less confusing

I think we should also use file naming conventions in order to avoid further problems [like this](https://www.w3schools.com/js/js_conventions.aspl)
 - In react the naming convention is UpperCamelCase
 - the rest of  javascript files. is adviced to use camelCase if im not mistaken.
 
Here's the final look after all of the above.

![image](https://user-images.githubusercontent.com/79209608/226103536-722e0161-3941-4651-900a-f7aa83926522.png)

of course feel free to tell whatever you disagree with :)

**Comment by RaduSeba** [comment]
Looks very good!

Gracias Sara.

**Comment by FdezAriasSara** [comment]
**IMPORTANT: index.js  needs to be outside any folder in /src , or npm start won't work**
